src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -0,0 +1,195 @@
+<h3>Bestandsliste</h3>
+
+@code {
+
+// Parameter Items for asset list
+    [Parameter] public string[] Items { get; set; }
```

Comment:
🔧 This should be a generic List and probably required.
```suggestion
    [Parameter] public required List<string> Items { get; set; }
```

Reply:
List is a powerful wrapper for the array type to get some extra features.
Arrays are only really used in some specific context